client/login/wp-login/index.jsx

```diff
@@ -135,10 +137,24 @@ class Login extends React.Component {
 				{ this.props.translate( 'Lost your password?' ) }
 			</a>;
 
+		const helpLink = <ExternalLink
+				icon={ true }
+				target="_blank"
+				href="http://en.support.wordpress.com/security/two-step-authentication/">
+				{ translate( 'Get help' ) }
+			</ExternalLink>;
+
+		if ( twoFactorAuthType ) {
```

Comment:
I may be misunderstanding this change but I believe we can make it more simply (with a single call to `compact`) but adding `twoFactorAuthType` to the `resetPasswordLink` and `showMagicLoginLink` conditions.